### ISSUE TITLE
Database table view error using Vitess

### ISSUE THREAD

**Issue by L3o-pold** [TablePlus/TablePlus#2941]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): MySQL

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): Version 5.3.0 (486)

3. The steps to reproduce this issue: Connect to a Vitess cluster, select any table on the left sidebar: "Error unsupported: '*' expression in cross-shard query"

<img width="459" alt="image" src="https://user-images.githubusercontent.com/4710495/217243537-30e6d3d0-2b60-4e47-b6bf-6848cc7fca04.png">

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
I found a similar issue here: https://github.com/vitessio/vitess/issues/9204 not sure if it is TablePlus issue or Vitess, but I think it is vitess issue because TablePlus has nothing todo with running the query.

**Comment by L3o-pold** [comment]
Cross shard join in the the same.

To avoid this issue Sequel Ace is doing the following queries to display the content of a table:

```sql
SHOW TABLE STATUS LIKE 'my_table';
SHOW CREATE TABLE `my_database`.`my_table`;
SELECT `id`, `name`, `other_field` FROM `my_table` LIMIT 0,1000;
```

**Comment by huyphams** [comment]
Ah I see. it uses the column list in the query instead of the *

**Comment by L3o-pold** [comment]
The `SELECT COUNT(*) as count FROM `my_database `.`my_table `;`  query that make TablePlus will not work either. Not sure how Sequel Ace is handling this.

**Comment by huyphams** [comment]
they would choose a random column such as primary key or just select count(0)

**Comment by L3o-pold** [comment]
it seems they use `SHOW TABLE STATUS LIKE 'my_table';` to fetch the `Rows` column.

**Comment by huyphams** [comment]
That query will show the estimated number, not the real-time count.

**Comment by huyphams** [comment]
It also not count the correct the number of rows with applied filter conditions.

**Comment by L3o-pold** [comment]
yeah Sequel Ace have no such information. But count(1) should do the trick.

**Comment by huyphams** [comment]
I feel like I should make a different driver profile for vitass because it has some differences from the original MySQL.

**Comment by L3o-pold** [comment]
Using the [schema tracking](https://vitess.io/docs/14.0/reference/features/schema-tracking/) feature that's now enabled by default, it work without any issue using the MySQL driver.

**Comment by L3o-pold** [comment]
Feel free to close if you feel that supporting a non default (since Vitess 14.0) configuration should not be supported.
